leshan-core/src/main/java/org/eclipse/leshan/core/node/codec/tlv/LwM2mNodeTlvDecoder.java

```diff
@@ -60,6 +79,21 @@ public <T extends LwM2mNode> T decode(byte[] content, LwM2mPath path, LwM2mModel
         }
     }
 
+    /**
+     * Get the Link parser used for parsing core links.
+     */
+    public LinkParser getLinkParser() {
+        return linkParser;
+    }
+
+    /**
+     * Set the Link parser used for parsing core links. If not set the
+     * {@link DefaultLwM2mLinkParser} is used.
+     */
+    public void setLinkParser(LinkParser linkParser) {
+        this.linkParser = linkParser;
+    }
```

Comment:
This doesn't compile because of : `The final field LwM2mNodeTlvDecoder.linkParser cannot be assigned`

I think we can remove the getter and setter.

Reply:
whoops, done